features/step_definitions/sriov.rb

```diff
@@ -54,6 +59,34 @@
   })
 end
 
+Given /^network-resources-injector daemon is ready$/ do
```

Comment:
@zhaozhanqi this step and the one for `admission webhook` daemon is almost identical...perhaps we can combine them into one?

Reply:
@pruan-rht thanks, I combine those two into one. 
```
Given /^(network-resources-injector|operator-webhook) daemon is ready$/ do | feature |
  ensure_admin_tagged
  project("openshift-sriov-network-operator")
  step %Q/3 pods become ready with labels:/, table(%{
    | app=#{feature} |
  })
end
```